pkg/cmd/testdata/scripts/importtarzstd.txtar

```diff
@@ -0,0 +1,13 @@
+[(darwin||openbsd||windows)] skip 'tar does not support zstd compression'
+[!exec:zstd] skip 'zstd not found in $PATH'
+exec tar -c -f archive.tar.zst --zstd archive
```

Comment:
This could be done with `exec tar -c -f - archive | zstd > archive.tar.zst` rather than expecting `--zstd`. macOS tar (`bsdtar`) *does* support `--use-compress-program`, so on macOS it could be `exec tar -c  --use-compress-program zstd -f archive.tar.zst archive`.

Reply:
Thanks, done.